src/interpreter/InterpretAdditionalItems.ts

```diff
@@ -0,0 +1,19 @@
+import { CommonModel } from 'models';
+import { Schema } from 'models/Schema';
+import { Interpreter, InterpreterOptions } from './Interpreter';
+
+/**
+ * Interpreter function for JSON Schema draft 7 additionalProperties keyword.
+ * 
+ * @param schema
+ * @param model
+ * @param interpreter
+ * @param interpreterOptions to control the interpret process
+ */
+export default function interpretAdditionalItems(schema: Schema, model: CommonModel, interpreter : Interpreter, interpreterOptions: InterpreterOptions = Interpreter.defaultInterpreterOptions): void {
+  if (model.type?.includes('array') === false) {return;}
+  const additionalItemsModel = interpreter.interpret(schema.additionalItems || true, interpreterOptions);
```

Comment:
`additionalItems` can be has `false` value, so this if isn't correct.

You should check first the `false` value:

```suggestion
  if (additionalItems === false) {return;}
  const additionalItemsModel = interpreter.interpret(schema.additionalItems || true, interpreterOptions);
```

Reply:
Good catch!

Gonna refactor it for both 🙂